avm/res/databricks/access-connector/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
-      "templateHash": "2158406963601048020"
+      "version": "0.24.24.22086",
```

Comment:
Hey @clintgrove, please use the latest version of bicep cli to regenerate readme and compiled json. That will allow publishing a new patch version of the module with the latest publishing feature "with source"

Reply:
Hi @eriqua  please review now and let me know if its good to go. I have upgraded to latest bicep and run it again